Docs: sync visualization docs with source (pt2) (#126813)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/gauge/index.md

| Style | Choose a gauge shape:<ul><li>**Circle** - Fills clockwise around a circle, starting from the 12 o'clock position.</li><li>**Arc** - Fills from left to right around an arc.</li></ul> |
| Orientation | Choose a stacking direction:<ul><li>**Auto** - Gauges display in rows and columns.</li><li>**Horizontal** - Gauges display top to bottom.</li><li>**Vertical** - Gauges display left to right.</li></ul> |

> | Gauge size | Choose a gauge size mode:<ul><li>**Auto** - Grafana determines the best gauge size.</li><li>**Manual** - Manually configure the gauge size.</li></ul>This option only applies when **Orientation** is set to **Horizontal** or **Vertical**. |

Reviewer: <img width="332" height="859" alt="Image" src="https://github.com/user-attachments/assets/e43796ac-1e81-4fbf-a499-a0323589548b" />

This threw me because gauge size only shows conditionally if the orientation is not `auto`. 

Same with min-height and min-width.

Probably not something we want to address in this PR but all these conditionals make it hard to consume to docs when comparing to an active viz. 

Wonder if we should look into a "docs mode" for viz where holding shift (for example) allows you to click on any setting and it takes you to the doc for that setting.

Author: Yeah, these conditionals make it hard to write the docs too, lol. Once we get all these in here, I can go through and try to at least make the "if this/then that" language more consistent so that the user can start to scan for it. 

Docs connected to a setting would be hella cool though!

| Min width | Set the minimum width of vertically-oriented gauges. If you set a minimum width, the x-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge size** is set to **Manual**. |

> | Min height | Set the minimum height of horizontally-oriented gauges. If you set a minimum height, the y-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge size** is set to **Manual**. |

Author: ```suggestion
| Min width | Set the minimum width of vertically-oriented gauges. If you set a minimum width, the x-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge orientation** is **Vertical** and **Gauge size** is set to **Manual**. |
| Min height | Set the minimum height of horizontally-oriented gauges. If you set a minimum height, the y-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge orientation** is **Horizontal** and **Gauge size** is set to **Manual**. |
```

| Bar width factor | Set a factor between 0.1 and 1 to control the width of the gauge bar relative to the total gauge area. |
| Bar style | Choose the shape of the gauge bar. Choose from **Flat** or **Rounded**. |

> | Segments | Enter a value between 1 and 100 to break the gauge into equal segments. Segments are always fully filled, even if the gauge's current value falls within a segment. |

Reviewer: Took me forever to find bar style, it looks like its only displayed if the segment count is zero.
Also looks like `bar style` should be after segments since it is conditional

<img width="333" height="583" alt="Image" src="https://github.com/user-attachments/assets/e163adb8-6eaa-4b11-84a9-96e299ac6064" />

```suggestion
| Bar style | Choose the shape of the gauge bar. Choose from **Flat** or **Rounded**. Only applies if Segments is set to one.  |
```

```suggestion
| Segments | Enter a value between 1 and 100 to break the gauge into equal segments. Segments are always fully filled, even if the gauge's current value falls within a segment. |
| Bar style | Choose the shape of the gauge bar. Choose from **Flat** or **Rounded**. Only applies if **Segments** is set to `1`. Note: All segmented gauges are flat. |
```

| Segment spacing | Enter a value between 0 and 1 to set the factor that controls the size of the gap between each segment. This option is only applies when the **Segments** value is more than `1`. |
| Text mode | Choose what text to render in the gauge:<ul><li>**Auto** - Grafana determines the text to show.</li><li>**Value and name** - Renders both the value and series name.</li><li>**Value** - Renders only the value.</li><li>**Name** - Renders only the series name.</li><li>**None** - Renders the gauge with no labels.</li></ul> |
| [Neutral value](#neutral-value) | Set the starting value from which every gauge will be filled. |
| [Show sparkline](#show-sparkline) | Toggle on the switch to render a sparkline containing the series data for the gauge. This option is on by default. |
| [Show thresholds](#show-threshold-markers) | Control whether a threshold band is shown outside the inner gauge value band. |
| Show labels | Control whether threshold and neutral labels are shown outside of the gauge. |

> | Endpoint marker | Choose how to mark the current value endpoint. Choose from **Point**, **Glow**, or **None**. **Glow** is available only in dark mode. |

Author: @fastfrwrd I can't actually see Endpoint marker in the UI. I'm not sure if the display of this is controlled by something else or if we got rid of this?
```suggestion
| Endpoint marker | Choose how to mark the current value endpoint. Choose from **Point**, **Glow**, or **None**. **Glow** is available only in dark mode. This option only applies when the **Bar style** is **Rounded**. |
```

Reviewer: Only shows when bar style is rounded (which only shows when segments == 1), so should display after "bar style".

<img width="354" height="254" alt="Image" src="https://github.com/user-attachments/assets/b32aa95e-d905-444a-9daa-3588a3c1b1d6" />

| Effects | Other styling choices you can apply to your gauge include: <ul><li>**Gradient** - Color transitions are represented with gradients for color schemes where the gauge color differs by value. This effect is on by default.</li><li>**Bar glow** - Adds a glowing shadow outside the gauge bar.</li><li>**Center glow** - The color representing the current gauge value is visible in the center of the gauge.</li></ul> |

<!-- prettier-ignore-end -->

#### Neutral value

Set the starting value from which every gauge will be filled when it shouldn't be the minimum value.
This option is especially useful in cases where the range of data values includes negative numbers.

> If you leave this empty, Grafana uses the configured minimum value.

Reviewer: This confused the heck outta me but the copy is correct, what was threw me is the behavior of min which is the min of the series and not zero by default.

### Neutral:
<img width="1017" height="492" alt="Image" src="https://github.com/user-attachments/assets/1ba247e5-d312-418a-aa90-866df8019ae9" />

### Min: 

<img width="1020" height="517" alt="Image" src="https://github.com/user-attachments/assets/4fe52501-27c2-4c60-9e8c-3e4dcd383e06" />

### Both:

#### Neutral lower then value:
<img width="1012" height="495" alt="Image" src="https://github.com/user-attachments/assets/0ce8c8f5-89e7-43d4-ba30-eb181035ce4a" />

#### Neutral higher then value:

<img width="1022" height="507" alt="Image" src="https://github.com/user-attachments/assets/5664a5c9-0273-4a0a-b3a6-c47bc29a5e6a" />

Author: Lemme know if you think the update to the copy helps solve the issue.

Reviewer 2: I had to re-read the sentences a few times to get the gist of it. Ran it through Opus and came up with this: 

"By default, a gauge starts filling from its minimum value rather than from zero. Use this option to set a different starting value. This is useful when your data includes negative numbers. If you leave it empty, Grafana uses the configured minimum value."

> Set the starting value from which every gauge will be filled when it shouldn't be the minimum value.

Author: ```suggestion
By default, the starting value of a gauge is the minimum value, not zero.
You can set the starting value from which every gauge will be filled when you don't want it to be the minimum value.
```

| Style | Choose a gauge shape:<ul><li>**Circle** - Fills clockwise around a circle, starting from the 12 o'clock position.</li><li>**Arc** - Fills from left to right around an arc.</li></ul> |
| Orientation | Choose a stacking direction:<ul><li>**Auto** - Gauges display in rows and columns.</li><li>**Horizontal** - Gauges display top to bottom.</li><li>**Vertical** - Gauges display left to right.</li></ul> |
| Gauge size | Choose a gauge size mode:<ul><li>**Auto** - Grafana determines the best gauge size.</li><li>**Manual** - Manually configure the gauge size.</li></ul>This option only applies when **Orientation** is set to **Horizontal** or **Vertical**. |
| Min width | Set the minimum width of vertically-oriented gauges. If you set a minimum width, the x-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge orientation** is **Vertical** and **Gauge size** is set to **Manual**. |

> | Min height | Set the minimum height of horizontally-oriented gauges. If you set a minimum height, the y-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge orientation** is **Horizontal** and **Gauge size** is set to **Manual**. |

Reviewer 2: Don't hyphenate compound words that end with -ly: developers.google.com/style/hyphens#exceptions_2
```suggestion
| Min width | Set the minimum width of vertically oriented gauges. If you set a minimum width, the x-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge orientation** is **Vertical** and **Gauge size** is set to **Manual**. |
| Min height | Set the minimum height of horizontally oriented gauges. If you set a minimum height, the y-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge orientation** is **Horizontal** and **Gauge size** is set to **Manual**. |
```

| [Show sparkline](#show-sparkline) | Toggle on the switch to render a sparkline containing the series data for the gauge. This option is on by default. |
| [Show thresholds](#show-threshold-markers) | Control whether a threshold band is shown outside the inner gauge value band. |
| Show labels | Control whether threshold and neutral labels are shown outside of the gauge. |
| Endpoint marker | Choose how to mark the current value endpoint. Choose from **Point**, **Glow**, or **None**. **Glow** is available only in dark mode. This option only applies when the **Bar style** is **Rounded**. |
| Effects | Other styling choices you can apply to your gauge include: <ul><li>**Gradient** - Color transitions are represented with gradients for color schemes where the gauge color differs by value. This effect is on by default.</li><li>**Bar glow** - Adds a glowing shadow outside the gauge bar.</li><li>**Center glow** - The color representing the current gauge value is visible in the center of the gauge.</li></ul> |

<!-- prettier-ignore-end -->

#### Neutral value

By default, the starting value of a gauge is the minimum value, not zero.

> You can set the starting value from which every gauge will be filled when you don't want it to be the minimum value.

Author: ```suggestion
You can change the starting value from which every gauge will be filled when you don't want it to be the minimum value.
```

| Gauge size | Choose a gauge size mode:<ul><li>**Auto** - Grafana determines the best gauge size.</li><li>**Manual** - Manually configure the gauge size.</li></ul>This option only applies when **Orientation** is set to **Horizontal** or **Vertical**. |

> | Min width | Set the minimum width of vertically-oriented gauges. If you set a minimum width, the x-axis scrollbar is automatically displayed when there's a large amount of data. This option only applies when **Gauge size** is set to **Manual**. |

Author: ```suggestion
| Min width | Set the minimum width of vertically-oriented gauges. If you set a minimum width, the x-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge size** is set to **Manual**. |
```

> | Min height | Set the minimum height of horizontally-oriented gauges. If you set a minimum height, the y-axis scrollbar is automatically displayed when there's a large amount of data. This option only applies when **Gauge size** is set to **Manual**. |

Author: ```suggestion
| Min height | Set the minimum height of horizontally-oriented gauges. If you set a minimum height, the y-axis scroll bar is automatically displayed when there's a large amount of data. This option only applies when **Gauge size** is set to **Manual**. |
```
